src/avalanchemq/launcher.cr

```diff
@@ -0,0 +1,233 @@
+require "file"
+require "systemd"
+
+module AvalancheMQ
+  class Launcher
+    @tls_context : (OpenSSL::SSL::Context::Server|Nil)
```

Comment:
Can add question mark instead of union of nil in crystal

Reply:
Ah, TIL